vendor/assets/stylesheets/spree/frontend/global.css.scss

```diff
@@ -1,7 +1,6106 @@
+
 @charset "utf-8";
 /* CSS Document */

```

Comment:
Why we replaced the whole file. As only new design classes needs to be add in this file. In this case we reverted the font icons code of previous file. Please discuss.